src/Evolution/Systems/GrMhd/ValenciaDivClean/FiniteDifference/Tag.hpp

```diff
@@ -40,12 +40,28 @@ struct Reconstructor : db::SimpleTag {
   static type create_from_options(
       const type& reconstructor,
       const ::evolution::dg::subcell::SubcellOptions& subcell_options) {
-    if (not subcell_options.finite_difference_derivative_order().has_value() and
+    if (static_cast<int>(subcell_options.finite_difference_derivative_order()) <
+            0 and
         not reconstructor->supports_adaptive_order()) {
       ERROR_NO_TRACE(
           "Cannot use adaptive finite difference derivative order with "
           "specified reconstructor.");
     }
+    if ((static_cast<int>(
+             subcell_options.finite_difference_derivative_order()) /
+             2 -
+         1) > static_cast<int>(reconstructor->ghost_zone_size())) {
```

Comment:
Remind me why the `if` condition isn't `(static_cast<int>(subcell_options.finite_difference_derivative_order()) - 1) / 2 > static_cast<int>(reconstructor->ghost_zone_size()`?  The logic in the original commit would only check for 1 ghost zone, for a 5th order WENOZ stencil when it should be 2.  Are the even order finite difference stencils different?

Reply:
Ah! Excellent question. For reconstruction we have:
```
o o | x x x
  ^   ^ ^ ^
``` 
where every point that has an `^` needs reconstruction, `o` is neighbor and `x` is ourselves. We need to do reconstruction in one neighbor element to get the neighbor's face solution on the boundary. For FD derivatives we just need:
```
o o | x x x
      ^ ^ ^
```
so we don't need to take the numerical derivative in our neighbor's cell, unlike reconstruction. This is why the FD derivatives are one stencil more compact that the reconstruction.

Does that answer your question?

Reply:
Yes it does; I was conflating the reconstruction and FD order.  So the FD order is for calculating spatial derivatives of quantities (e.g., the metric variables) to be used in (e.g.,) source terms, whereas reconstruction stencil/order (e.g., WENOZ) is specified through the reconstructions scheme?

Reply:
Almost/mostly! This controls the order of the `d_i F^i` FD derivative. Currently the metric terms always use the highest derivative possible based on the FD reconstruction stencil size. In practice that means for 2nd-order reconstruction you do 4th order FD derivatives for the metric, 5th order recons gives 6th-order metric, etc.

Compared to FV, this would be controlling the order of the Gaussian quadrature used to compute `int n_i F^i`. In FD you change the FD derivative order.

Does that make sense?